ecl/hql/hqlattr.cpp

```diff
@@ -308,6 +308,7 @@ unsigned getOperatorMetaFlags(node_operator op)
     case no_normalize:
     case no_distributed:
     case no_preservemeta:
+    case no_unordered:
```

Comment:
The attribute was removed to ensure that ORDERED(bool) was treated consistently.  The unordered node doesn't have an ordered counterpart, so it makes sense for it to be a no_unordered.